.circleci/config.yml

```
@@ -164,8 +157,9 @@ jobs:
     parallelism: 2
     working_directory: /home/qa/code
     docker:
-      # - image: lecaoquochung/docker-images:puppeteer-v1.4.0
-      - image: ghcr.io/lecaoquochung/docker-images:puppeteer-latest-build
+      - image: ghcr.io/lecaoquochung/docker-images:puppeteer-latest-build # Dev
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)